tf2_eigen_kdl/CHANGELOG.rst

```diff
@@ -0,0 +1,14 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
For a backport, just leave out the CHANGELOG.rst; that will get added as part of the release process.

Reply:
Removed in a4f3f4b35350a0368c7f89589daa434cf80ae618